workshop/workshop_instructions1.md

```diff
@@ -1,5 +1,7 @@
 # Satellite Workshop Instructions: CI/CD with GitHub Actions
 
+Workflow 1 will focus on creating a CI workflow for our [app](Need link to this). At the end of this exercise we will learn a) How to create a new CI workflow b) About Workflow templates c) Using UI Action authoring Intellisense d) Configuring Action triggers and type of triggers e) About Job strategies f) Publishing to Packages  
```

Comment:
@kaverma - I think we will need one doc just to describe what your App is. Can we create one for it and link that in this instruction readme as well.

Reply:
sure